README.md

```diff
@@ -196,6 +196,10 @@ And pass in a valid API key (if needed) in the way that provider expects.
 
 To redact api keys, modify the `filter_headers` list in the `get_vcr` function in `ddapm_test_agent/vcr_proxy.py`. This can be confirmed by viewing cassettes in the `vcr-cassettes` directory (or the otherwise specified directory), and verifying that any new cassettes do not contain the api key.
 
+#### Running in CI
```

Comment:
should we just reuse the existing CI_MODE variable? Might make sense to not introduce another flag unless necessary so that future users don't accidentally miss it and face the same issue we did

Reply:
followed up oob - gonna keep with the separate ci mode variable for now, we can maybe coalesce the two snapshot & vcr variables down the line